.github/workflows/build_osgeo4w.sh

```diff
@@ -1,145 +1,22 @@
 #!/bin/sh
 #
-# Usage: build_osgeo4w.sh [-p] [path]
+# Usage: build_osgeo4w.sh [path]
 #
 # By default, the script will look for the source code in the current directory
 # and create bin.x86_64-w64-mingw32\grass$ver.bat (run this batch file to start
 # GRASS GIS) and dist.x86_64-w64-mingw32\etc\env.bat.
 #
-# -p	optionally install GRASS GIS to C:\OSGeo4W64\opt\grass (run
```

Comment:
IIRC @HuidaeCho used **-p** to have zippable binaries.

Reply:
Yes, I used `-p` to create an unzippable package, which is needed in a non-admin environment. Most of this original workflow script is now moved to `mswindows/osgeo4w/build_osgeo4w.sh`. What happened to the old OSGeo4W build script that was in `mswindows/osgeo4w`?

Reply:
> Yes, I used `-p` to create an unzippable package, which is needed in a non-admin environment. 

Thanks for the info. I wasn`t aware of that. Will see that this is preserved...

> Most of this original workflow script is now moved to `mswindows/osgeo4w/build_osgeo4w.sh`. What happened to the old OSGeo4W build script that was in `mswindows/osgeo4w`?

There is still `package.sh` in `mswindows/osgeo4w`. I think, in future it would be good to have one build recipe that is used for both packaging and testing (and thus lives in `./mswindows/` and not `.github/workflows/` (unless we have a packaging workflow)). The build parts could then be removed from `package.sh` and just called...